examples/plot_ucurve.py

```diff
@@ -17,8 +17,8 @@
 from skglm import Lasso
 
 # %%
-# First, we load the dataset and keep 2000 features.
-# We also retrain 2000 samples in training dataset.
+# First, we load the dataset and keep ``2000`` features.
```

Comment:
thanks @Badr-MOUFAD for the update, I'm just removing the code style here since it's not a variable and having too much of those make reading more difficult IMO